test/utils/IexecWrapper.ts

```diff
@@ -89,6 +104,23 @@ export class IexecWrapper {
             .then((tx) => tx.wait());
     }
 
+    /**
+     * Hash an order using current domain.
+     */
+    hashOrder(order: Record<string, any>) {
+        return hashOrder(this.domain, order);
+    }
+
+    /**
+     * Sign an order operation using current domain.
+     */
+    async signOrderOperation(
+        orderOperation: OrderOperation,
+        signer: SignerWithAddress,
+    ): Promise<void> {
+        return signOrderOperation(this.domain, orderOperation, signer);
+    }
```

Comment:
Same question

Reply:
And there:
https://github.com/iExecBlockchainComputing/PoCo/blob/82bfa22f1935ccc547d1ff34e6f404f5cfd104d7/test/byContract/IexecOrderManagement/IexecOrderManagement.ts#L173
:slightly_smiling_face: